### PR TITLE
🌐 Update Chinese translation for `docs/zh/docs/contributing.md`

### PR DESCRIPTION
**Pull request by Aruelius**
中英文同步

sync the English version

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit 60dac472da4a9a2db749dfdbfda1ca1c24496d0d at: https://5f2d8cf6.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 5ccc37d8e41f1f2f7346480c030b6333e40aa9ea at: https://0869ddb8.fastapitiangolo.pages.dev

File: docs/zh/docs/contributing.md [hunk 1]
```diff
@@ -423,16 +393,15 @@ theme:
...
 Building docs for: es
 Successfully built docs for: es
-Copying en index.md to README.md
 ```
 
 </div>
 
+这样会对每一种语言构建一个独立的文档站点，并最终把这些站点全部打包输出到 `./site/` 目录。
+
 这将在 `./docs_build/` 目录中为每一种语言生成全部的文档。还包括添加所有缺少翻译的文件，并带有一条"此文件还没有翻译"的提醒。但是你不需要对该目录执行任何操作。
```
Comment (HiemalBeryl) [inline docs/zh/docs/contributing.md:405]:
```suggestion
```

File: docs/zh/docs/contributing.md [hunk 2]
```diff
@@ -423,16 +393,15 @@ theme:
...
 
 然后，它针对每种语言构建独立的 MkDocs 站点，将它们组合在一起，并在 `./site/` 目录中生成最终的输出。
```
Comment (HiemalBeryl) [inline docs/zh/docs/contributing.md:407]: 这里和上面重复了，英文原文里也没有找到这句
```suggestion
```

**Review by HiemalBeryl — APPROVED** [review]

**Comment by tiangolo** [comment]
📝 Docs preview for commit 1ae53d4ca5757b9509b7df19266879e460bde24f at: https://0db0e4cb.fastapitiangolo.pages.dev

**Comment by alejsdev** [comment]
Thanks for the help! @Aruelius @HiemalBeryl 🚀✨

**Pull request merged**